weather_data_stack/infra/airbyte/main.tf

```diff
@@ -0,0 +1,54 @@
+// Airbyte Terraform provider documentation: https://registry.terraform.io/providers/airbytehq/airbyte/latest/docs
+
+// Sources
+resource "airbyte_source_weatherstack" "my_source_weatherstack" {
+  configuration = {
+    credentials = {
+      source_weatherstack_authentication_access_key = {
+        access_key = var.access_key
+
+      }
+
+    }
+
+
+    query = var.query
+
+    historical_date = "2023-09-28"
+
+  }
+  name         = "Weatherstack"
+  workspace_id = var.workspace_id
+}
```

Comment:
@Ronlin1 does this actually work? I don't see the `airbyte_source_weatherstack` listed in the terraform documentation. If it doesn't work you can just use the Airbyte UI to create the connectors, but in that case, please remove all the references to Terraform from the quickstart

Reply:
No, as @bishalbera said previously yes its not listed in the terraform docs. I only used the airbyte ui to create the connectors . so should I just delete the infra/airbyte file entirely as it involves terraform

Reply:
Verify, if am good to go

Reply:
Hi @ThaliaBarrera , waiting on you.

Reply:
The PR is merged @Ronlin1!